packages/main/files/package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "@google/generative-ai/files",
+  "name": "@google/generative-ai-files",
   "description": "GoogleAI file upload manager",
-  "main": "./dist/files/index.js",
-  "browser": "./dist/files/index.mjs",
-  "module": "./dist/files/index.mjs",
-  "typings": "./dist/files/files.d.ts"
+  "main": "../dist/files/index.js",
```

Comment:
Would this break the module imports?

Reply:
Module imports? I realized it might affect the rollup config as it gets paths from here, so I changed that.

Reply:
I tested this with tsconfig set to module:commonjs, module:node16 and in a vite bundler proejct and it seems to work in all cases.